samples/highcharts/chart/events-fullscreenopen-fullscreenclose/demo.css

```diff
@@ -0,0 +1,3 @@
+#container {
```

Comment:
I think you should use shorter folder names. What about
```samples/highcharts/chart/fullscreen-events/...```
?

Also, do we have a folder for fullscreen demos? (does it have to be in the `/chart/` folder?)

Reply:
Ok